src/node_data.js

```diff
@@ -110,23 +116,24 @@ const initData = function(node, nodeName, key) {
  * @return {!NodeData} The NodeData for this Node.
  */
 const getData = function(node) {
-  let data = node['__incrementalDOMData'];
-
-  if (!data) {
-    data = importNode(node);
+  if (!node[DATA_PROP]) {
```

Comment:
Updated to just call `importNode`. Guess we are getting closer to making `getData` a [simple getter](https://github.com/google/incremental-dom/pull/257#discussion_r63462018). 😉